backend/src/relation-requests/relation-requests.service.ts

```diff
@@ -24,55 +25,52 @@ export class RelationRequestsService {
   //  MUTATION
   //**************************************************//
   async create(
-    userSenderId: string,
-    userReceiverId: string
+    data: RelationRequestsInput
   ): Promise<RelationRequests | RelationFriend> {
     const userFriend = await this.relationFriendService.isFriend(
-      userSenderId,
-      userReceiverId
+      data.userSenderId,
+      data.userReceiverId
     )
     const userBlocked = await this.relationBlockedService.isBlocked(
-      userSenderId,
-      userReceiverId
+      data.userSenderId,
+      data.userReceiverId
     )
     const userBlockedYou = await this.relationBlockedService.isBlocked(
-      userReceiverId,
-      userSenderId
+      data.userReceiverId,
+      data.userSenderId
     )
     const userRequestReceived = await this.isRequested(
-      userReceiverId,
-      userSenderId
+      data.userReceiverId,
+      data.userSenderId
     )
-    if (userSenderId === userReceiverId) {
+    if (data.userSenderId === data.userReceiverId) {
       throw new ExceptionRequestingYourself()
     }
 
-    // if blocked
     if (userBlocked) {
       throw new ExceptionUserBlocked()
     }
 
-    // if blocked by the other user
     if (userBlockedYou) {
       throw new ExceptionUserBlockedYou()
     }
 
-    // if friend
     if (userFriend) {
       throw new ExceptionUsersAlreadyFriend()
     }
 
-    // if requestFriendReceived
     if (userRequestReceived) {
-      await this.delete(userReceiverId, userSenderId)
-      return this.relationFriendService.create(userSenderId, userReceiverId)
+      await this.delete(data.userReceiverId, data.userSenderId)
+      return this.relationFriendService.create(
+        data.userSenderId,
+        data.userReceiverId
+      )
     }
 
-    // if no relations
     return this.prisma.relationRequests.create({
```

Comment:
should be:
return this.prisma.relationRequests.create({data})
if you have to write
data: {
        userSenderId: data.userSenderId,
        userReceiverId: data.userReceiverId
      }
for it to work then there is a problem